globus_sdk/transfer/data.py

```diff
@@ -1,6 +1,6 @@
 """
 Data helper classes for constructing Transfer API documents. All classes should
-extend ``dict``, so they can be passed seemlesly to
+extend ``dict``, so they can be passed seamlesly to
```

Comment:
needs more `s`

Reply:
Haha, yes, typo fixing should actually fix the typos...